src/Serilog.Sinks.PeriodicBatching/Sinks/PeriodicBatching/BoundedConcurrentQueue.cs

```diff
@@ -1,36 +1,45 @@
-﻿using System;
+// Copyright 2013-2020 Serilog Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
 using System.Collections.Concurrent;
 using System.Threading;
 
 namespace Serilog.Sinks.PeriodicBatching
 {
-    class BoundedConcurrentQueue<T> 
+    class BoundedConcurrentQueue<T>
     {
-        const int NON_BOUNDED = -1;
+        public const int NonBounded = -1;
```

Comment:
Are we tracking to rename this to `Unbounded` on the next release? `NonBounded` is not proper English.

Reply:
Can't believe I didn't catch that when writing it.

Reply:
:-) I fixed the XML parameter docs but forgot about this; I'll send one though (though it's not public API surface)